src/components/Calendar/Calendar.vue

```diff
@@ -44,12 +44,37 @@
         <template v-if="columnsProp">
           <slot name="columnSelectionHeader"></slot>
           <button
+            v-if="allColumnsUnchecked"
             style="padding-left: 3.75rem; padding-right: 3.75rem"
             class="btn btn-secondary m-2"
             @click="columns.forEach(c => (c.checked = true))"
           >
             Alle auswählen
           </button>
+          <button
+            v-if="allColumnsChecked"
+            style="padding-left: 3.75rem; padding-right: 3.75rem"
+            class="btn btn-secondary m-2"
+            @click="columns.forEach(c => (c.checked = false))"
+          >
+            Alle abwählen
+          </button>
+          <div class="d-flex flex-column" v-if="!allColumnsUnchecked && !allColumnsChecked">
+            <button
+              style="padding-left: 3.75rem; padding-right: 3.75rem"
+              class="btn btn-secondary ms-2 m-2"
+              @click="columns.forEach(g => (g.checked = true))"
+            >
+              Alle auswählen
+            </button>
+            <button
+              style="padding-left: 3.75rem; padding-right: 3.75rem"
+              class="btn btn-secondary ms-2 m-2"
+              @click="columns.forEach(g => (g.checked = false))"
+            >
+              Alle abwählen
+            </button>
+          </div>

```

Comment:
i think you dont need to duplicate the buttons if you write the v-if directly on the button and keep the wrapping div with d-flex